src/main/java/com/alysaa/geyseradmintools/listeners/AdminLockChat.java

```diff
@@ -5,17 +5,14 @@
 import org.bukkit.event.player.AsyncPlayerChatEvent;
 
 public class AdminLockChat implements Listener {
+
     public static boolean isMuted = false;
+
     @EventHandler
     public void onChat(AsyncPlayerChatEvent e) {
-        if(isMuted) {
+        if (isMuted) {
             e.setCancelled(true);
-        }
-        if(e.getPlayer().hasPermission("geyseradmintools.gadmin")) {
-            e.setCancelled(false);
-        }
-        else {
-            e.setCancelled(false);
+            e.getPlayer().sendMessage("shhhhhhhhh!");
         }
```

Comment:
Not sure if this was suppose to be some bypass thing but everything I removed did nothing